src/pages/questionAnswerViewer/subComponents/EditQuestion.jsx

```diff
@@ -41,13 +41,13 @@ export default function EditQuestion({ question, onUpdated, onDeleted }) {
     if (!newQuestion.metadata.name) {
       return;
     }
-    await API.updateQuestion(newQuestion, token);
-    onUpdated();
+    const response = await API.updateQuestion(newQuestion, token);
```

Comment:
I think we need to handle errors when updating and deleting questions and answers. I'm testing by actually killing the robokache server and then trying these different operations to see what happens and I'm getting confusing messages.